tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineMongoDBAdminBundle\Tests\DependencyInjection;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineMongoDBAdminBundle\DependencyInjection\Configuration;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+use Symfony\Component\Config\Definition\Processor;
+
+class ConfigurationTest extends TestCase
```

Comment:
IMO it is better to test the configuration using: https://github.com/SymfonyTest/SymfonyConfigTest we have some examples on sonata

Reply:
Done!